src/collectors/eventstoreprojections/eventstoreprojections.py

```diff
@@ -0,0 +1,100 @@
+# coding=utf-8
+
+"""
+A Simple collector which calls the Eventstore API for
+the status of projections and parses this to flat metrics.
+String values are ignored, except for the name and status.
+Name is used for the metric path, status is translated to
+an integer (running = 1, stopped = 0).
+Note: "$" are replaced by underscores (_) in the projection
+name to avoid problems with grafana.
+
+This collector is based upon the HTTPJSONCollector.
+
+#### Dependencies
+
+ * urllib2
+
+"""
+
+import urllib2
+import json
+import diamond.collector
+
+
+class EventstoreProjectionsCollector(diamond.collector.Collector):
+
+    def get_default_config_help(self):
+        config_help = super(
+            EventstoreProjectionsCollector, self).get_default_config_help(
+        )
+        config_help.update({
+            'url': 'Full URL',
+            'headers': 'Header variable if needed. '
+            'Will be added to every request',
```

Comment:
Could you please keep the lines of the description at the same indent level? This might require wrapping it in parenthesis.